### DISCUSSION TITLE
Tanya seputar Apache Kafka

### DISCUSSION THREAD

**Discussion by Ninoambara** [deaafrizal/forum#13] [Troubleshooting Room]
Jadi saya dapet projectan dimana disana mengimplementasikan apache kafka dimana apache kafka digunaan untuk perantara untuk microservice pengiriman email saya, ada yang tau ga yaa kaya gimana cara buatnya di typescript

**Comment by hansputera** [comment]
Mungkin abangnya bisa coba kulik2 library/wrapper Apache Kafka di NodeJS - https://kafka.js.org/

Di web tersebut sudah ada beberapa samples code yang bisa dicoba sesuai usecasenya (entah itu menjadi producer/consumer)

Contoh penggunaannya ada pada: https://github.com/maddymanu/TypeScript-KafkaJS-Example
